src/CartFee.php

```diff
@@ -23,12 +24,14 @@ class CartFee
      *
      * @param $amount
      * @param $taxable
+     * @param $tax
      * @param array $options
      */
-    public function __construct($amount, $taxable, $options = [])
+    public function __construct($amount, $taxable = false, $tax = 0, $options = [])
```

Comment:
not sure $tax = 0 is needed, since we have taxable = true or false we can then set the tax based on that.